rtf_vis_tool/src/Components/frustum_classes/se3.js

```diff
@@ -51,6 +51,22 @@ class SE3 {
                             point_world_coords.get(0,2)]])
         return point_world_coords;
     }
+
+    inverse() {
+        /* Computes the inverse of the current pose matrix. (Since numjs doesn't have a function for it)
+
+        Returns:
+            A new SE3 object storing the inverse of the current pose matrix.
+        */
+        const Rt = this.rotation.T;
+        const t_negative = nj.array([-1 * this.translation.get(0), 
```

Comment:
is it not possible to just multiply the whole array by -1, instead of doing it on a per-element basis?

Reply:
Unfortunately, `numjs` does not support broadcasting, so it won't allow me to multiply an array by a scalar.

Reply:
https://github.com/nicolaspanel/numjs/issues/1

Reply:
cool, nevermind then